clang/cmake/caches/Standalone_libc_baremetal.cache

```diff
@@ -0,0 +1,43 @@
+# Expects target triple to be passed as `ARCH_TRIPLE`
+message(STATUS "--- Discovered target triple: ${ARCH_TRIPLE} ---")
```

Comment:
Maybe reword this? It doesn't sound like we're really discovering anything here, just printing out what the user pass along.